package.json

```diff
@@ -1,9 +1,10 @@
 {
     "name": "@space-metaverse-ag/space-ui",
-    "version": "0.2.8",
+    "version": "0.2.11",
     "module": "./index.js",
     "types": "./index.d.ts",
     "dependencies": {
+        "framer-motion": "^7.6.1",
```

Comment:
I had to `--force` to get this package to install. Has broken peer deps.
If not too hard, would prefer it done in-house to prevent bloating our packages. Adds 2mb.

Reply:
Makes sense, I will remove framer-motion

At first I would like to apply it, but I will bring the animations with transitions and keyframes, to make the bundle lighter